cmd/beacon-chain/flags/base.go

```diff
@@ -334,4 +334,9 @@ var (
 		Usage: "Specifies the retention period for the pruner service in terms of epochs. " +
 			"If this value is less than MIN_EPOCHS_FOR_BLOCK_REQUESTS, it will be ignored.",
 	}
+	// SubscribeAllColumns enables subscription to all blob columns.
+	SubscribeAllColumns = &cli.BoolFlag{
```

Comment:
I would personnally prefer to have a more generic options like: "subscribe-all-sidecar-subnets".
This way, this option ensures that, whatever the type of sharded sidecars we use (data columns for peerDAS, cells for fullDAS), we subscribe to all needed subnets to download all shards.